### PR TITLE
Support for Drawing API

### LINKED ISSUE #28: Drawing API support
@tomchentw 
Is there any support for the [Google Maps drawing API](https://developers.google.com/maps/documentation/javascript/drawinglayer) via this project?

This is **crucial** for me, so if it's not built in, is there a way I can get a reference to the underlying Google Maps `map` object so I can point a `DrawingManager` I instantiate at it?

### PR DESCRIPTION
**Pull request by idolize**
Resolves #28 and exposes DrawingManager as `require("react-google-maps").DrawingManager`

### CODE REVIEW

File: src/internals/SimpleChildComponent.js [hunk 1]
```diff
@@ -32,7 +33,16 @@ class SimpleChildComponent extends EventComponent {
       }
       instance.setOptions(googleMapsConfig);
     } else {
-      const GoogleMapsClass = googleMapsApi[this.constructor._GoogleMapsClassName];
+      const googleMapsClassName = this.constructor._GoogleMapsClassName;
+      if (!objectPath.has(googleMapsApi, googleMapsClassName)) {
```
Comment (tomchentw) [inline src/internals/SimpleChildComponent.js]: `object-path` looks like a good library. Thanks!

**Comment by tomchentw** [comment]
@idolize This looks really promising and runs well. Thanks for the hard work.
However, could you `amend` the commit messages to [follow the conventiona-changelog](https://github.com/ajoslin/conventional-changelog/blob/master/CONVENTIONS.md) spec? Use:

```log
feat(DrawingManager): Support for Drawing API
```

and

```log
feat(SimpleChildComponent): Add warning text

 * if user attempts to use componenet without corresponding library
```

File: examples/gh-pages/scripts/components/drawing/DrawingTools.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            ]
+          }}
+          circleOptions={{
+            fillColor: '#ffff00',
+            fillOpacity: 1,
+            strokeWeight: 5,
+            clickable: false,
+            editable: true,
+            zIndex: 1
+          }}>
```
Comment (tomchentw) [inline examples/gh-pages/scripts/components/drawing/DrawingTools.js]: You've forgot to close the tag here: `}}>`

**Comment by tomchentw** [comment]
After fixing the close tag issue, it runs well on my computer. :) Congrats @idolize !

screen shot 2015-05-14 at 10 27 51 pm

**Comment by idolize** [comment]
:+1: Thanks - should be good now

**Comment by tomchentw** [comment]
Great!

**Pull request merged**